events/2024-06-10-mtb-ngs.html

```diff
@@ -14,15 +14,15 @@
 date_end: 2024-06-14 # optional, if event is more than one day
 
 contributions:
-  organisers: # GTN contributors or funders, must be defined in CONTRIBUTORS.yaml
-  - dbrites
-  instructors:
+  organisers &  instructors: # GTN contributors or funders, must be defined in CONTRIBUTORS.yaml
```

Comment:
@dbrites this will not be valid. You can either put everybody as organizers, or everybody as instructors. It is also possible to put people both under organisers and under instructors (e.g. you?). So whatever you prefer.